aspnetcore-webapi/RateLimitingDotNET8/RateLimitingDotNET8/Policies.cs

```diff
@@ -0,0 +1,9 @@
+﻿namespace RateLimitingDotNET8;
+
+public static class Policies
+{
+    public static string Fixed = "fixed";
+    public static string Sliding = "fixed";
+    public static string Token = "token";
+    public static string Concurrency = "Concurrency";
```

Comment:
Since all values are lowercase, we can have "Concurrency" also in lowercase here